brjs-core/src/main/java/org/bladerunnerjs/model/NodeImporter.java

```diff
@@ -205,22 +205,30 @@ private static void findAndReplaceInTextFiles(BRJS brjs, Collection<File> files,
 	{
 		for (File f : files) {
 			if (f.length() != 0) {
-				if (checkFileMimeType(f).startsWith("text")) {
-					findAndReplaceInTextFile(brjs, f, sourceRequirePrefix, targetRequirePrefix);
+				try {
+					if (checkFileMimeType(f).startsWith("text")) {
+						findAndReplaceInTextFile(brjs, f, sourceRequirePrefix, targetRequirePrefix);
+					}
+				}
+				catch (MagicMatchNotFoundException e) {
```

Comment:
Is this not going to prevent property files etc. from being filtered? Maybe we should filter if theres a `MagicMatchNotFoundException`.